npm/src/browser.ts

```diff
@@ -167,3 +167,4 @@ export { log, type LogLevel };
 export type { ICompilerWorker, ICompiler };
 export type { ILanguageServiceWorker, ILanguageService };
 export { type LanguageServiceEvent } from "./language-service/language-service.js";
+export { PackageType } from "../lib/web/qsc_wasm.js";
```

Comment:
Being we import all of the wasm package as 'wasm' in line 7 above, you should just be able to export `wasm.PackageType as PackageType` in the exports on line 166.

Reply:
I can't get that syntax to work; I can export `wasm` itself but then callers have to import that and use `wasm.PackageType.Lib` in the code.